### ISSUE TITLE
This image is missing a text alternative (`alt` attribute). This is a problem for people using screen readers.

### ISSUE THREAD

**Issue by kadirselcuk** [kadirselcuk/next-auth#81]
This image is missing a text alternative (`alt` attribute). This is a problem for people using screen readers.

_Originally posted by @accesslint[bot] in https://github.com/kadirselcuk/next-auth/pull/57#discussion_r584178027_
